edx-platform/pearson-ironwood/lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -61,8 +61,8 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://image.freepik.com/vector-gratis/vector-onda-azul-transparente_1055-7084.jpg" width="70"
-                                    height="30" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
+                                    src="https://mylearningx.pearson.com/static/pearson-pols-theme/images/logo.png" width="40"
+                                    height="40" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
```

Comment:
Why height and not width? @luismorenolopera

Reply:
both values ​​are assigned

Reply:
Sorry, I didn't notice them.